### DISCUSSION TITLE
ability to pass list of included attributes

### DISCUSSION THREAD

**Discussion by sundeep-amrute-bfef** [tiangolo/pydantic-sqlalchemy#81] [Questions] [answered]
Similar to the exclude parameter of sqlalchemy_to_pydantic(), it would be useful to be alternately able to specify an include list of columns. This would be convenient when we have a model with say 30 columns of which only 10 need to be specified when creating a new item. This would be similar to the Marshmallow "only" option.

**Comment by a-atalla** [comment]
fro reference, it is implemented in this fork 
https://github.com/ggabriel96/alchemista/

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile:

This would be a feature request.

I deprecated this library a while ago in favor of SQLModel which would solve all the same use cases in a much better way, and I'm directing all the effort in that direction.

SQLModel uses Pydantic models to define both data and table (SQL) models, so it's easier to do all these things. :rocket:
